src/main/io/serial_cli.c

```diff
@@ -970,6 +973,61 @@ static void cliSerial(char *cmdline)
 
 }
 
+static void cliSerialPassthrough(char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        cliShowParseError();
+        return;
+    }
+
+    int id = -1;
+    unsigned baud = 0;
+    char* tok = strtok(cmdline, " ");
+    int index = 0;
+
+    while (tok != NULL) {
+        switch(index) {
+            case 0:
+                id = atoi(tok);
+                break;
+            case 1:
+                baud = atoi(tok);
+                break;
+        }
+        index++;
+        tok = strtok(NULL, " ");
+    }
+
+    serialPort_t *passThroughPort;
+    serialPortUsage_t *passThroughPortUsage = findSerialPortUsageByIdentifier(id);
+    if (!passThroughPortUsage || passThroughPortUsage->serialPort == NULL) {
+        if (!baud)
+        {
+            printf("Port %d is not open, you must specify baud\r\n", id);
+            return;
+        }
+
+        passThroughPort = openSerialPort(id, FUNCTION_PASSTHROUGH, NULL,
+                                         baudRates[baud], MODE_RXTX,
```

Comment:
It is probably better to avoid baudrate table lookup completely or allow user to specify non-default baudrate.
```
if(baud<BAUD_RATE_COUNT)
   baud = baudRates[baud];
```